app/views/fines/index.html.erb

```diff
@@ -56,25 +56,25 @@
         </a>
 
         <ul class="dropdown-menu" aria-labelledby="dropdownMenuLink">
-          <li class="dropdown-item active" data-sort="due_date">Reason</li>
-          <li class="dropdown-item" data-sort="status">Date paid</li>
-          <li class="dropdown-item" data-sort="title">Title</li>
+          <li class="dropdown-item" data-sort="bill_description">Reason</li>
+          <li class="dropdown-item active" data-sort="payment_date">Date paid</li>
+          <li class="dropdown-item" data-sort="item_title">Title</li>
         </ul>
       </div>
     </div>
 
     <div class="d-none d-md-flex row font-weight-bold list-header">
       <div class="row col-md-12 col-lg-6">
-        <div class="col-md-4" data-sort="reason">Reason</div>
-        <div class="col-md-4" data-sort="amount">Amount</div>
-        <div class="col-md-4 active" data-sort="due_date">Paid on</div>
+        <div class="col-md-4" data-sort="bill_description">Reason</div>
+        <div class="col-md-4">Amount</div>
```

Comment:
I think we'd want to make amount sortable too? It'd be a little weird for one column header not to be, right @jvine ?